### PR TITLE
CM-653: How to Force Run .NET Application as Administrator

### PR DESCRIPTION
**Pull request by Emmanuel-Adom**

### CODE REVIEW

File: dotnet-projects/HowToForceRunDotNetApplicationAsAdministrator/AdministratorChecker.cs
Comment (fl0tschi) [inline dotnet-projects/HowToForceRunDotNetApplicationAsAdministrator/AdministratorChecker.cs:1]: I suppose this one is a leftover, looks like you moved it to the project folder

File: dotnet-projects/HowToForceRunDotNetApplicationAsAdministrator/IAdministratorChecker.cs
Comment (fl0tschi) [inline dotnet-projects/HowToForceRunDotNetApplicationAsAdministrator/IAdministratorChecker.cs:1]: Same here, I think you moved the code to the project folder, please remove this one.

File: dotnet-projects/HowToForceRunDotNetApplicationAsAdministrator/AdministratorApp/Program.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{
+    static void Main()
+    {
+        IAdministratorChecker administratorChecker = new AdministratorChecker();
+        bool isAdmin = administratorChecker.IsCurrentUserAdmin();
+
+        PrivilegeStatusPrinter printer = new();
+
+        printer.PrintPrivilegeStatus(isAdmin);
+
```
Comment (fl0tschi) [inline dotnet-projects/HowToForceRunDotNetApplicationAsAdministrator/AdministratorApp/Program.cs:13]: New line can be removed

File: dotnet-projects/HowToForceRunDotNetApplicationAsAdministrator/AdministratorChecker.cs
Comment (Emmanuel-Adom) [reply dotnet-projects/HowToForceRunDotNetApplicationAsAdministrator/AdministratorChecker.cs:1]: Oh yes sorry, its a leftover, will remove it. Thank you @fl0tschi.

File: dotnet-projects/HowToForceRunDotNetApplicationAsAdministrator/IAdministratorChecker.cs
Comment (Emmanuel-Adom) [reply dotnet-projects/HowToForceRunDotNetApplicationAsAdministrator/IAdministratorChecker.cs:1]: Thank you, will remove it.

**Pull request closed without merging**